test/Reflection/conformance_descriptors.swift

```diff
@@ -5,6 +5,7 @@
 //
 // Temporarily disable on AArch64 Linux (rdar://88451721)
 // UNSUPPORTED: OS=linux-gnu && CPU=aarch64
+// XFAIL: OS=linux-android
```

Comment:
Same comment, do you have the failure handy? The radar is for a different failure mode, but I can try to confirm that it's the same issue.

Reply:
[Same here](https://github.com/swiftlang/swift/pull/82325#issuecomment-3002137848)